packages/math/src/pool.ts

```diff
@@ -75,11 +77,12 @@ export const convertDollarValueToCoins = (
 };
 
 export const convertDollarValueToShares = (
+  osmosisAssets: OsmosisAsset[],
```

Comment:
same as above, let's use `Asset[]` or `AssetList[]` from chain-registry/types and avoid making new types

Reply:
ok no problem